e2e/scenario/types.go

```diff
@@ -24,6 +24,15 @@ type Config struct {
 	BootstrapConfigMutator func(*testing.T, *datamodel.NodeBootstrappingConfiguration)
 	VMConfigMutator        func(*armcompute.VirtualMachineScaleSet)
 	Validator              func(context.Context, *testing.T, *ValidationInput) error
+	LiveVMValidators       []*LiveVMValidator
```

Comment:
nit: would be nice if our "validator interface" was either consistently a func or array of funcs

Reply:
maybe not really a big deal. I actually like the separate validator funcs for reuse.